src/core.js

```diff
@@ -1,3 +1,4 @@
+import * as d3 from 'd3';
```

Comment:
I've added D3 as a global/external to the Rollup instantiation; AFAICT, that's the intended way of using external modules with Rollup if you don't want to include them within the bundle itself. See also: https://github.com/rollup/rollup/wiki/JavaScript-API#globals